File: frontend/src/types/store.ts
export interface StoreListItemDto {
  id: number
  code: string
  name: string
  contactName?: string
  contactPhone?: string
  isEnabled: boolean
  createdAt: string
}

export interface SaveStoreRequest {
  code: string
  name: string
  contactName?: string
  contactPhone?: string
  isEnabled: boolean
}

File: backend/src/FaJuan.Api/Domain/Entities/Store.cs
namespace FaJuan.Api.Domain.Entities;

public class Store
{
    public long Id { get; set; }
    public string Code { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string? ContactName { get; set; }
    public string? ContactPhone { get; set; }
    public bool IsEnabled { get; set; } = true;
    public DateTime CreatedAt { get; set; } = DateTime.Now;
}
